src/main/java/uk/nhs/digital/nhsconnect/nhais/mapper/FromFhirToEdifact.java

```diff
@@ -0,0 +1,31 @@
+package uk.nhs.digital.nhsconnect.nhais.mapper;
+
+import org.hl7.fhir.r4.model.Parameters;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.Segment;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FromFhirToEdifact {
+    //TODO fix mapper order if needed
+    private final List<FromFhirToEdifactMapper<?>> MAPPERS = Arrays.asList(
```

Comment:
Or inject a list of `FromFhirToEdifactMapper<?>` but I'm not sure how to handle ordering of them then

Reply:
Because leaving it as is, introduces the danger of shared state

Reply:
atm it's not a risk as this class is instantiated by hand in https://github.com/nhsconnect/integration-adaptor-nhais/pull/57/files#diff-944e71f1d57df8b0086adc7678283210R77 but please add a TODO comment here that it need's refactoring